packages/sdk/src/unlist/README.md

```diff
@@ -0,0 +1,34 @@
+[//]: # `{ "title": "unlist", "order": 0.9 }`
+
+# Unlist 
+
+Calling this method revokes token approval from the supplied market address resulting in the unlisting of the provided token.
+
+Token approvals are permissions granted to decentralized apps to access tokens in your crypto wallet.
+
+This means that the marketplace won't be able to list your token for sale because they won't be able to transfer it once its sold.
```

Comment:
It is already listed. And the method also removes the listing right? So any calls to `buy` will fail, because the market no longer knows of the listings existence.

Reply:
I wrote this to explain why they would need to grant access to their token. I think when people read that they don't instantly rationalise why a marketplace would need this